src/components/ChartMenu/Sidebar/BoardItem/BoardItem.tsx

```diff
@@ -0,0 +1,27 @@
+import styles from "@components/ChartMenu/Sidebar/BoardItem/BoardItem.module.scss";
+import { Board } from "@models/PodData/Board";
+import PacketItem from "@components/ChartMenu/Sidebar/PacketItem/PacketItem";
+import { TreeNode } from "@components/ChartMenu/TreeNode";
+type Props = {
+  name: string;
+  packetNodes: TreeNode;
+};
+
+export const BoardItem = ({ name, packetNodes }: Props) => {
+  return (
+    <div id={styles.wrapper}>
+      <div id={styles.header}>{name}</div>
+      <div id={`${styles.packetList} ${styles.treeNode}`}>
+        {Object.entries(packetNodes).map(([name, measurementNodes]) => {
```

Comment:
move this to a new method, u should not mix logic and view:

```
const generatePacketItems = (packetNodes): JSX.Element => (
    Object.entries(packetNodes).map(([name, measurementNodes]) => (
            <PacketItem
              key={name}
              name={name}
              measurementNodes={measurementNodes!} />
    )
)

export const BoardItem = ({ name, packetNodes }: Props): JSX.Element => (
    <div id={styles.wrapper}>
      <div id={styles.header}>{name}</div>
      <div id={`${styles.packetList} ${styles.treeNode}`}>
       { generatePacketItems(packetNodes) }
       </div>
    </div>
)
```

It is much more readable